polybar-scripts/network-openwrt-status/README.md

```diff
@@ -0,0 +1,28 @@
+# Script: network-openwrt-status
+
+Print stats from OpenWrt router. Such as signal levels, sms count and traffic bandwidth.
+
+Written for Microdrive Tandem-4GT-OEM modem, but can be adopted to work with all OpenWrt routers.
+
+![screenshot](screenshots/1.png)
```

Comment:
You uploaded the same screenshot file twice. Could you check this?

Reply:
yes, I know. but they have the same output. I just don't use huawei modem at the moment. but if you insist, I can make an actual screenshot.

Reply:
fixed

Reply:
Awesome.